php_script_as_jpg_metadata_file_upload_breach/Ressources/Explanation.md

```diff
@@ -59,7 +59,7 @@ Upload
 Either from the HTML form fields or from the request body, we can find the `form-data` entries:
 
 - `uploaded` (contains the file)
-- `MAX_FILE_SIZE` (the maximum size of the file; not safe and can be changed)
+- `MAX_FILE_SIZE` (the maximum size of the file; either not safe as can be changed)
```

Comment:
I'm not sure either should be put in this sentence.

Reply:
Indeed